runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -345,7 +344,16 @@ private void calculate()
 				actionCount = (int) Math.ceil(neededXP / xp);
 			}
 
-			slot.setText("Lvl. " + action.getLevel() + " (" + formatXPActionString(xp, actionCount, "exp) - "));
+			if (isBuyableSkill(currentSkill))
+			{
+				slot.setText("<html>Lvl. " + action.getLevel() + " (" + formatXPActionString(xp, actionCount, "exp) - ") + formatXPActionCostString(actionCount, action.getName()));
+
+			}
+			else
+			{
+				slot.setText("Lvl. " + action.getLevel() + " (" + formatXPActionString(xp, actionCount, "exp) - "));
+
+			}
```

Comment:
This can use some de-duplication.
```suggestion
			String slotText = "Lvl. " + action.getLevel() + " (" + formatXPActionString(xp, actionCount, "exp) - "));

			if (isBuyableSkill(currentSkill))
			{
				slotText = "<html>" + slotText + formatXPActionCostString(actionCount, action.getName());
			}

			slot.setText(slotText);
```